ftest.sh

```diff
@@ -147,8 +148,8 @@ $NFS_SERVER:$PWD $DAOS_BASE nfs defaults 0 0 # added by ftest.sh
 wq
 EOF
 sudo mount $DAOS_BASE
-rm -rf /tmp/Functional_$TEST_TAG/
-mkdir -p /tmp/Functional_$TEST_TAG/
+rm -rf $TEST_DIR/
+mkdir -p $TEST_DIR/
```

Comment:
Both of these lines above will raise a [shellcheck](https://www.shellcheck.net/) [SC2086: Double quote to prevent globbing and word splitting](https://github.com/koalaman/shellcheck/wiki/SC2086) error at some point in the future when our shellcheck version catches up to the latest.


 shellcheck SC2086: Double quote to prevent globbing and word splitting. error at some point in the future when our shellcheck version catches up to the latest.

Reply:
Thanks for the great feedback.  Fix applied.